common-libs/internal-api-clients/package.json

```diff
@@ -43,8 +42,9 @@
   "homepage": "https://github.com/affinityproject/affinidi-core-sdk#readme",
   "dependencies": {
     "@affinidi/affinidi-did-auth-lib": "^1.0.8",
```

Comment:
can we make the version 1.0.8 until we solve the issues with the did auth lib versioning?

Reply:
Will this solve the issue?

Reply:
Fixed version at 1.0.8